view/theme/frio/templates/wall_thread.tpl

```diff
@@ -404,7 +404,12 @@ as the value of $top_child_total (this is done at the end of this file)
                                                 <a class="btn-link navicon ignore" href="javascript:ignoreAuthor('item/ignore/{{$item.id}}/{{$item.return}}', 'item-{{$item.guid}}');" title="{{$item.ignore_author.ignore}}"><i class="fa fa-ban" aria-hidden="true"></i> {{$item.ignore_author.ignore}}</a>
                                         </li>
                                         {{/if}}
-                                    </ul>
+                                        {{if $item.collapse}}
+										<li role="menuitem">
+												<a class="btn-link navicon collapse" href="javascript:collapseAuthor('item/collapse/{{$item.id}}/{{$item.return}}', 'item-{{$item.guid}}');" title="{{$item.collapse.collapse}}"><i class="fa fa-ban" aria-hidden="true"></i> {{$item.collapse.collapse}}</a>
+										</li>
+										{{/if}}
+									</ul>
```

Comment:
Please normalize the indentation in this file. The rest of the file uses spaces.